apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -218,6 +224,11 @@ const CustomizableCurriculumCatalogCard = ({
     }
   };
 
+  const studentButtonColor =
```

Comment:
Since this button is viewable regardless of if they're a student, could this variable be renamed to something like `quickViewButtonColor` or something?